contrib/utilities/jsontomarkdown.py

```diff
@@ -3,6 +3,25 @@
 import json
 from sys import argv, exit, stderr
 
+
+def print_if_parameter_or_alias(entry, path_str, true_name, cur_path, output_file) :
+    if "value" in entry:
+        # this is a parameter
+        print("(parameters:" + path_str + ")=", file=output_file)
```

Comment:
This line is still duplicated. Do you want to move it outside the ``if``?

Reply:
I was about to do it, but then realized that it's really one part of a two-line statement whereby the first writes the label and the second the section heading. I think that conceptually they should stay together.

Reply:
OK.